applications/RANSApplication/python_scripts/formulations/utilities.py

```diff
@@ -0,0 +1,244 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+import KratosMultiphysics.RANSApplication as KratosRANS
+
+from KratosMultiphysics import IsDistributedRun
+from KratosMultiphysics import VariableUtils
+from KratosMultiphysics.kratos_utilities import CheckIfApplicationsAvailable
+
+from KratosMultiphysics.RANSApplication import RansVariableUtilities
+
+if (IsDistributedRun()
+        and CheckIfApplicationsAvailable("TrilinosApplication")):
+    from KratosMultiphysics.TrilinosApplication import trilinos_linear_solver_factory as linear_solver_factory
+    from KratosMultiphysics.TrilinosApplication import TrilinosMixedGenericCriteria as residual_criteria
+    from KratosMultiphysics.TrilinosApplication import TrilinosNewtonRaphsonStrategy as newton_raphson_strategy
+    from KratosMultiphysics.RANSApplication.block_builder_and_solvers import TrilinosPeriodicBlockBuilderAndSolver as periodic_block_builder_and_solver
+    from KratosMultiphysics.RANSApplication.block_builder_and_solvers import TrilinosBlockBuilderAndSolver as block_builder_and_solver
```

Comment:
Why not importing them as usual? This is inconsistent with the other fluid applications and requires introducing an extra file.

Reply:
BlockBuilders and solvers have inconsistent OMP and MPI Ctors, so If I am to remove `block_builder_and_solvers.py`, then I need to have `if` conditions in `CreateResidualBasedBlockBuilderAndSolver` method. I wanted to have all the if conditions at one place, so this file is consistent and clean.

Reply:
removed :)